Makefile

```diff
@@ -18,11 +18,16 @@ install:
 	cp misc/aw-server.service /usr/lib/systemd/user/aw-server.service
 
 test:
-	python3 -c 'import aw_server'
+	pipenv run python3 -c 'import aw_server'
```

Comment:
Having python installed does not necessarily mean that the user has pipenv installed, so if you want to change this default you should at least make sure that "pip3 install pipenv" is run before this.

Personally I prefer not using pipenv, so this also breaks my development workflow. But I guess for most people it's a good thing though.

Reply:
It's to ensure that we always test with the exact-by-hash versions specified in the Pipfile.lock, so needed for reproducible builds.